packages/plugin-ext/src/plugin/tree/tree-views.ts

```diff
@@ -247,11 +247,14 @@ class TreeViewExtImpl<T> implements Disposable {
 
     async reveal(element: T, options?: Partial<TreeViewRevealOptions>): Promise<void> {
         await this.pendingRefresh;
+        const select = !(options?.select === false); // default to true
+        const focus = !!options?.focus;
+        const expand = typeof options?.expand === 'undefined' ? false : options!.expand!;
```

Comment:
minor: the trailing `!` does not seem necessary?

```suggestion
        const expand = typeof options?.expand === 'undefined' ? false : options!.expand;
```